### ISSUE TITLE
Loops: Unhandled Exception Caught!

### ISSUE THREAD

**Issue by winkelchri** [CGCookie/retopoflow#671]
Just wanted to add a loop cut into a PolyStrip.

-------------------------------------

```
Environment:

- RetopoFlow: 2.0.1
- Blender: 2.79.0 master 2018-03-22
- Platform: Windows, 10, 10.0.17134, AMD64, AMD64 Family 23 Model 1 Stepping 1, AuthenticAMD
- GPU: NVIDIA Corporation, GeForce GTX 1080 Ti/PCIe/SSE2, 4.6.0 NVIDIA 416.94, 4.60 NVIDIA
- Timestamp: 2018-11-29 23:55:48.080078
- Undo: 

Error Hash: eeb8abcd493b29b7d42c825bc06be46e

Trace:

- EXCEPTION (<class 'TypeError'>): 'NoneType' object is not iterable
- .../rfmode\rfcontext_target.py
- 000 0090:get_vis_accel() self.accel_vis_verts = { bmv for bmv in self.accel_vis_verts if bmv.is_valid }
- .../common\profiler.py
- 001 0174:wrapper() return fn(*args, **kwargs)
- .../rfmode\rfcontext_target.py
- 002 0113:accel_nearest2D_edge() vis_accel = self.get_vis_accel()
- .../common\profiler.py
- 003 0174:wrapper() return fn(*args, **kwargs)
- .../rfmode\rftool_loops.py
- 004 0072:set_next_state() self.nearest_edge,_ = self.rfcontext.accel_nearest2D_edge(max_dist=10)
- .../common\profiler.py
- 005 0174:wrapper() return fn(*args, **kwargs)
- .../rfmode\rftool_loops.py
- 006 0109:modal_main() self.set_next_state()
- .../rfmode\rftool.py
- 007 0110:modal() nmode = self.FSM[self.mode]()
- 008 0114:modal() raise e     # passing on the exception to RFContext
- .../rfmode\rfcontext.py
- 009 0577:modal_main() self.tool.modal()
- 010 0490:modal() nmode = self.FSM[self.mode]()
```

**Comment by winkelchri** [comment]
![2018-11-29 23_57_17-window](https://user-images.githubusercontent.com/11255205/49257436-9333fc80-f432-11e8-9a69-8350974d9217.png)

**Comment by vxlcoder** [comment]
Thanks for the report, @winkelchri!

This issue is related to issue #669, which has been fixed in branch `release/2.0.2` and will be released _very_ soon.  (feel free to check out `release/2.0.2` now if you'd like, though)

**Comment by winkelchri** [comment]
Okay. Cool. Thank you for the quick response. Great product by the way!
